www/index.html

```diff
@@ -80,7 +80,9 @@
 		});
 		return uuid;
 	}
-	document.getElementById('boxurl').value = 'https://jsonbox.io/box_' + create_UUID();
+	var base = window.location.protocol + '//' + window.location.hostname;
+	base += (window.location.port === 80 || window.location.port) === 443 ? '' : ':' + window.location.port;
```

Comment:
There is an issue with this implementation. I get an extra `:` in the URL. 
Sample `https://jsonbox.io:/box_129326b6cacf71fb5f14`.